py-scripts/lf_interop_ping.py

```diff
@@ -486,6 +492,39 @@ def get_pass_fail_list(self, os_type):
                     pass_fail_list.append("FAIL")
             self.pass_fail_list = pass_fail_list
 
+    def add_live_view_images_to_report(self, report, report_path):
```

Comment:
Please add type annotations for the new function. Welcome to add to the docstring, but not required.